apps/web/src/components/Composer/Actions/SpaceSettings/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { LockClosedIcon } from '@heroicons/react/outline';
+import { Modal, Toggle, Tooltip } from '@lenster/ui';
+import { t, Trans } from '@lingui/macro';
+import { motion } from 'framer-motion';
+import type { FC, ReactNode } from 'react';
+import React, { useState } from 'react';
+import { useSpacesStore } from 'src/store/spaces';
+
+import TokenGateForm from './TokenGateForm';
+
+interface SpaceSettingsProps {
+  children?: ReactNode;
+}
+
+const SpaceSettings: FC<SpaceSettingsProps> = ({ children }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  const { isRecordingOn, setIsRecordingOn } = useSpacesStore();
+
+  return (
+    <div>
```

Comment:
use fragment instead of div, divs make dom huge 😕

Reply:
Updated